packages/web/src/scss/components/Breadcrumbs/README.md

```diff
@@ -3,20 +3,32 @@
 ```html
 <nav aria-label="Breadcrumb" class="Breadcrumbs">
   <ol>
-    <li class="d-none d-tablet-flex">
-      <Link href="#rootUrl" color="primary" isUnderlined>Root</Link>
```

Comment:
Why did you remove `Link` component and added simple `a` back again? I thought in README should be code that is like an example - or is it because this block code is wrapped in html block style?

Reply:
I removed it because this is the web package Readme. There is no Link component.